src/ThemedStyleSheet.js

```diff
@@ -83,6 +83,15 @@ function flush() {
   }
 }
 
+// Exported until we deprecate this API completely
+// eslint-disable-next-line no-underscore-dangle
+export function _getInterface() {
+  return styleInterface;
+}
+
+// Exported until we deprecate this API completely
+export { get as _getTheme };
+
```

Comment:
This is so we can fall back to these singletons from the contextual implementation.